MechTransfer.csproj

```diff
@@ -10,4 +10,9 @@
   <ItemGroup>
     <PackageReference Include="tModLoader.CodeAssist" Version="0.1.*" />
   </ItemGroup>
+  <ItemGroup>
+    <Reference Include="MagicStorage">
+      <HintPath>..\ModAssemblies\MagicStorage_v0.6.0.3.dll</HintPath>
```

Comment:
Consider using `MagicStorage_v0.6.*.dll`, so that the reference still works if MagicStorage is updated.

Reply:
This one is not an option. This reference exists for visual studio to be able to compile. I wonder, though... Is it only important on my end? If so i'll just start excluding this change from my commits
One thing i can assure you is that tModLoader doesn't care about it

Reply:
I get that a reference is necessary to compile the mod, I'm just saying the exact Magic Storage version shouldn't be hard-coded here. But I can fix that later.

Reply:
No if i set it to * it doesn't recognize the file. It's a reference to the exact file on my computer (which means i probably shouldn't have included it but uhh too late i guess)
The actual reference data used by tmodloader is in build.txt